google/cloud/bigquery_storage_v1/reader.py

```diff
@@ -81,14 +83,12 @@ class ReadRowsStream(object):
     method to parse all messages into a :class:`pandas.DataFrame`.
     """
 
-    def __init__(self, wrapped, client, name, offset, read_rows_kwargs):
+    def __init__(
+        self, client, name, offset, read_rows_kwargs, retry_delay_callback=None
```

Comment:
Technically a breaking change, though I don't expect anyone to be constructing a ReadRowsStream directly, except perhaps in unit tests.

We should have had this before, but please add a comment to this class's docstring like the following:

```
    This object should not be created directly, but is returned by other
    methods in this library.
```

(Pulled from https://github.com/googleapis/python-pubsub/blob/main/google/cloud/pubsub_v1/futures.py)

Reply:
Also, it's not clear to me why we need to make this breaking change to begin with.

Reply:
Added the comment.

I removed the wrapped argument because ReadRowsStream assumes it will always get a valid stream with it. However, the RESOURCE_EXHAUSTED error can be the very first thing returned from the bq storage api server. If we keep the old api, we need to handle RESOURCE_EXHAUSTED errors in two different places. With this change we only need to handle it in one place.